### PR TITLE
Fix redraw for system roles, move software proposal and use real registration widget

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: package/yast2-installation.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Fri Jan 27 09:11:32 UTC 2017 - jreidinger@suse.com
+
+- Respect newly registered update repos in software proposal and
```
Comment (mvidner) [inline package/yast2-installation.changes]: += Do register.

**Comment by mvidner** [comment]
@jreidinger I don't understand why this PR apparently undoes the error handling that you added in #502

**Comment by mvidner** [comment]
Aaah, it uses `validate` now!

**Comment by jreidinger** [comment]
@mvidner yes, I use validate as more standard way to do it and also no need to redraw and redraw again when you click it

**Comment by mvidner** [comment]
The code looks good. Let me test it with a DUD.

**Comment by jreidinger** [comment]
I test it expect registration widget.

**Comment by mvidner** [comment]
Thanks!
But it's still failing because it is calling the older User widget API that does not expect the additional flag we're passing.

**Pull request merged**